components/user-store/org.wso2.carbon.identity.user.store.configuration.ui/src/main/resources/web/userstore_config/userstore-config.jsp

```diff
@@ -760,27 +778,31 @@
                             if (propertyName != null && propertyName.trim().length() > 0) {
 
                         %>
-                        <td class="leftCol-med" width="50%" style="text-align:left;" id="<%=name%>"><%=propertyName%>
+                        <td class="leftCol-med" width="50%" style="text-align:left;"
+                        id="<%=Encode.forHtmlAttribute(name)%>"><%=Encode.forHtml(propertyName)%>
                         </td>
-                        <input type="hidden" name=<%=name%> id=<%=name%> value="<%=advancedProperties[x].getName()%>"/>
+                        <input type="hidden" name=<%=Encode.forHtmlAttribute(name)%>
+                        id=<%=Encode.forHtmlAttribute(name)%>
+                        value="<%=Encode.forHtmlAttribute(advancedProperties[x].getName())%>"/>
                         </td>
                         <td style="width:30%" style="text-align:left;">
                             <%
                                 if (propertyValue != null) {
                                     if (isBoolean == 1) { %>
-                            <input type="checkbox" name=<%=value%> id=<%=value%>
-                                   class="checkbox" checked/>
+                            <input type="checkbox" name=<%=Encode.forHtmlAttribute(value)%>
+                            id=<%=Encode.forHtmlAttribute(value)%> class="checkbox" checked/>
 
                             <%
                             } else if (isBoolean == 0) { %>
-                            <input type="checkbox" name=<%=value%> id=<%=value%>
-                                   class="checkbox"/>
+                            <input type="checkbox" name=<%=Encode.forHtmlAttribute(value)%>
+                            id=<%=Encode.forHtmlAttribute(value)%> class="checkbox"/>
```

Comment:
variable value is hardcoded and does not contain special characters. No need to encode that variable